pubspec.yaml

```diff
@@ -47,6 +47,7 @@ dependencies:
   go_router: ^13.2.0
   equatable: ^2.0.5
   loading_animation_widget: 1.2.0+4
+  flutter_launcher_icons: ^0.13.1

```

Comment:
This package seems useful to me when the logos are different between platforms.
If we use the same icons for all platforms, I think we can do without it, and for Android, I also suggest that you replace the "ic_launcher.png" files with the logo, so the settings do not have to be changed.